ft_atoi.c

```diff
@@ -10,20 +10,24 @@
 /*                                                                            */
 /* ************************************************************************** */
 
-int	ft_atoi(const char *str)
+int ft_atoi(const char *str)
 {
 	int sign;
 	int res;
 
 	res = 0;
 	sign = 1;
-	while (*str == ' ' || *str == '\f' || *str == '\r' || *str == '\t'
-			|| *str == '\v' || *str == '\n')
+	while (*str == ' ' ||
```

Comment:
I'm not sure if the norminette will be okay with this, you can use "42 norminette" extension in VSCode